### PR TITLE
Fix selenium in prod

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?

The selenium tests are broken in PROD. This behaviour was caused because in https://github.com/guardian/support-frontend/pull/218 the domain of the `_test_usename` cookie is tied to `support.theguardian.com` instead of `.theguardian.com`. This makes that contributions doesn't realise that it should use PayPal sandbox instead of PayPal PROD which make the test fail. This behaviour was not spotted in DEV because contribution uses PayPal sandbox by default in DEV. 

This PR sets correctly the domain of `_test_username` to `.theguardian.com`. In addition, the timeout is increased to bear in mind the fact that in the recurring flow we now wait 30 seconds up most for the step functions.

## Changes

* Fix cookie
* Increased timeout

### CODE REVIEW

**Comment by svillafe** [comment]
Sets GU_DOMAIN in Travis ✅

File: test/selenium/util/Driver.scala [hunk 1]
```diff
@@ -1,6 +1,9 @@
 package selenium.util
 
 import java.net.URL
+import java.util
```
Comment (davidfurey) [inline test/selenium/util/Driver.scala]: Redundant import

File: test/selenium/util/Config.scala [hunk 2]
```diff
@@ -10,6 +10,8 @@ object Config {
 
   private val conf = ConfigFactory.load()
 
+  val guDomain = conf.getString("guDomain")
```
Comment (davidfurey) [inline test/selenium/util/Config.scala]: I should probably change https://github.com/guardian/support-frontend/blob/master/app/controllers/TestUsersManagement.scala#L18 to use this new property.

No real need to shorten this to `gu`, `guardianDomain` would probably be clearer for new people.

Comment (svillafe) [reply test/selenium/util/Config.scala]: All the comments are addressed here: f72a26799fb356af300fbd8facfb12976d8d730f . Please, could you double check that the wiring part is correct?

**Comment by jacobwinch** [comment]
This works now when we run it in PROD (by requesting a build for our branch with the TravisAPI):
https://travis-ci.org/guardian/support-frontend/builds/275415293
